### PR TITLE
M kovalsky/alttext title subtitle

### PR DESCRIPTION
**Pull request by m-kovalsky**

### CODE REVIEW

File: src/sempy_labs/report/_report_helper.py [hunk 1]
```diff
@@ -252,3 +252,59 @@ def find_entity_property_pairs(data, result=None, keys_path=None):
...
+
+def _get_agg_type_mapping() -> dict:
+
+    schema_url = "https://developer.microsoft.com/json-schemas/fabric/item/report/definition/semanticQuery/1.2.0/schema.json"
+    response = requests.get(schema_url)
+    schema = response.json()
+    aggtypes_schema = schema.get("definitions", {}).get("QueryAggregateFunction", {})
+
+    agg_type_map = {}
+    agg_type_map = {
```
Comment (eisber) [inline src/sempy_labs/report/_report_helper.py:265]: can you add an example of what you are extracting

File: src/sempy_labs/report/_report_helper.py [hunk 2]
```diff
@@ -252,3 +252,59 @@ def find_entity_property_pairs(data, result=None, keys_path=None):
...
+            expr_json.get("Aggregation", {})
+            .get("Expression", {})
+            .get("Column", {})
+            .get("Property", "Column not found")
+        )
+        function_id = expr_json.get("Aggregation", {}).get("Function", "-1")
+        function = agg_type_map.get(function_id)
+        expr = f"{function}('{entity}'[{column}])"
+    elif expr_type == "Measure":
+        entity = (
```
Comment (eisber) [inline src/sempy_labs/report/_report_helper.py:299]: what's the use of entity here?

File: src/sempy_labs/report/_report_helper.py [hunk 3]
```diff
@@ -252,3 +252,59 @@ def find_entity_property_pairs(data, result=None, keys_path=None):
...
+            .get("SourceRef", {})
+            .get("Entity", "Entity not found")
+        )
+        column = (
+            expr_json.get("Aggregation", {})
+            .get("Expression", {})
+            .get("Column", {})
+            .get("Property", "Column not found")
+        )
+        function_id = expr_json.get("Aggregation", {}).get("Function", "-1")
```
Comment (eisber) [inline src/sempy_labs/report/_report_helper.py:295]: what's the expectation if neither entity and/or column doesn't resolve? throw an exception? just put a string into the outcome? if users want to filter, they'd have to search for "Column not found"

**Review by eisber — CHANGES_REQUESTED** [review]

File: src/sempy_labs/report/_report_helper.py (hunk 1 above)
Comment (m-kovalsky) [reply src/sempy_labs/report/_report_helper.py:265]: added

File: src/sempy_labs/report/_report_helper.py (hunk 2 above)
Comment (m-kovalsky) [reply src/sempy_labs/report/_report_helper.py:299]: removed

File: src/sempy_labs/report/_report_helper.py (hunk 3 above)
Comment (m-kovalsky) [reply src/sempy_labs/report/_report_helper.py:295]: updated these to unknown

**Review by eisber — APPROVED** [review]

**Pull request merged**